modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ServiceFlavor.java

```diff
@@ -34,4 +34,7 @@ public class ServiceFlavor implements Serializable {
     @Schema(description = "The properties of the flavor")
     private Map<String, String> properties;
 
+    @NotNull
+    @Schema(description = "The priority of the flavor. The larger value means higher priority.")
```

Comment:
it must be other way round. "The larger the value means lower priority"

Reply:
Updated.